### PR TITLE
Update Release data for CR3

### LINKED ISSUE #6912: Fix CR3 data in release3.json
### Expected Behavior

release3.json data is correct and consistent

### Current Behavior

A number of things seem off:
* StableToken
  * Missing a value for “symbol”
  * initialBalanceAddresses should be a list
  * initialBalanceValues should be a list
* Exchange
  * Missing a value for “stableToken”
  * Spread is 0.0005, this should be 0.005 to be consistent with cUSD
  * Reserve fraction is 0.01, this should be 0.005 (for now) to be consistent with cUSD (according to celocli network:parameters)
  * minimumReports is 1, this should be 5 to be consistent with cUSD

### PR DESCRIPTION
**Pull request by martinvol**
Values were wrong and some were not even valid.

For stableTokenEUR:
```
string calldata _name = "Celo Euro"
string calldata _symbol = "cEUR"
uint8 _decimals = "18" # matches cUSD
address registryAddress = "0x000000000000000000000000000000000000ce10"
uint256 inflationRate = "1" (FixidityLib) # matches cUSD
uint256 inflationFactorUpdatePeriod = "47304000" # matches cUSD, `1.5 * YEAR`
address[] calldata initialBalanceAddresses = ["0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972", "0x989e1a3B344A43911e02cCC609D469fbc15AB1F1"]
uint256[] calldata initialBalanceValues = ["2000000000000000000000000", "6000000000000000000000000"]
string calldata exchangeIdentifier = "ExchangeEUR" # This has to match the registry, otherwise it won't be able to mint
```
For exchangeEUR:
```
address registryAddress = "0x000000000000000000000000000000000000ce10"
address stableToken = "0x0000000000000000000000000000000000000000" # should be set by an upcoming cEUR activation governance proposal, meanwhile exchangeEUR will remain unusable.
uint256 _spread = "0.005" # (fixidity) matches cUSD value
uint256 _reserveFraction = "0.0013" # (fixidity) a conservative value with preliminary analysis, can be updated by an upcoming cEUR activation governance proposal. For comparison, cUSD is now at 0.005
uint256 _updateFrequency = "300" # (in seconds) matches cUSD value, can be updated by an upcoming cEUR activation governance proposal
uint256 _minimumReports = "5" # matches cUSD value, can be updated by an upcoming cEUR activation governance proposal
```

All the parameters marked with `fixidity` need to be multiplied by `10^24`, like [shown here](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0008.md).

Fixes #6912

### CODE REVIEW

File: packages/protocol/releaseData/initializationData/release3.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", "0x0000000000000000000000000000000000000000", "0", "ExchangeEUR"],
-  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", "0.0005", "0.01", "300", "1"]
-}
\ No newline at end of file
+  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", ["0x0000000000000000000000000000000000000000"[, ["0"], "ExchangeEUR"],
```
Comment (martinvol) [inline packages/protocol/releaseData/initializationData/release3.json]:
```suggestion
  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", ["0x0000000000000000000000000000000000000000"], ["0"], "ExchangeEUR"],
```

File: packages/protocol/releaseData/initializationData/release3.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", "0x0000000000000000000000000000000000000000", "0", "ExchangeEUR"],
-  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", "0.0005", "0.01", "300", "1"]
-}
\ No newline at end of file
+  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", ["0x0000000000000000000000000000000000000000"], ["0"], "ExchangeEUR"],
+  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", null, "0.005", "0.0013", "300", "5"]
```
Comment (yorhodes) [inline packages/protocol/releaseData/initializationData/release3.json]:
```suggestion
  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", "0x0000000000000000000000000000000000000000", "0.005", "0.0013", "300", "5"]
```

**Review by yorhodes — CHANGES_REQUESTED** [review]
I think we want to put in the pre allocation here right? https://forum.celo.org/t/initial-allocation-of-celo-euro/761

File: packages/protocol/releaseData/initializationData/release3.json [hunk 3]
```diff
@@ -1,4 +1,4 @@
 {
-  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", "0x0000000000000000000000000000000000000000", "0", "ExchangeEUR"],
-  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", "0.0005", "0.01", "300", "1"]
-}
\ No newline at end of file
+  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", ["0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972", "0x0000000000000000000000000000000000000000"], ["2000000000000000000000000", "4000000000000000000000000"], "ExchangeEUR"],
```
Comment (martinvol) [inline packages/protocol/releaseData/initializationData/release3.json]: Update with foundation address

Comment (martinvol) [inline packages/protocol/releaseData/initializationData/release3.json]:
```suggestion
  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", ["0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972", "0x989e1a3B344A43911e02cCC609D469fbc15AB1F1"], ["2000000000000000000000000", "4000000000000000000000000"], "ExchangeEUR"],
```

File: packages/protocol/releaseData/initializationData/release3.json [hunk 4]
```diff
@@ -1,4 +1,4 @@
 {
-  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", "0x0000000000000000000000000000000000000000", "0", "ExchangeEUR"],
-  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", "0.0005", "0.01", "300", "1"]
-}
\ No newline at end of file
+  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", ["0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972", "0x989e1a3B344A43911e02cCC609D469fbc15AB1F1"], ["2000000000000000000000000", "4000000000000000000000000"], "ExchangeEUR"],
```
Comment (martinvol) [inline packages/protocol/releaseData/initializationData/release3.json]: 2M cEUR for the community fund at [0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972](https://explorer.celo.org/address/0xd533ca259b330c7a88f74e000a3faea2d63b7972/transactions)
5M cEUR to the foundation at [0x989e1a3B344A43911e02cCC609D469fbc15AB1F1](https://explorer.celo.org/address/0x989e1a3b344a43911e02ccc609d469fbc15ab1f1/transactions)

Comment (asaj) [reply packages/protocol/releaseData/initializationData/release3.json]: You have 4M in the code. And I think we said 6M in the governance call

Comment (asaj) [inline packages/protocol/releaseData/initializationData/release3.json]: `inflationRate` is a fixidity value please fix

File: packages/protocol/releaseData/initializationData/release3.json [hunk 5]
```diff
@@ -1,4 +1,4 @@
...
+  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", "0x0000000000000000000000000000000000000000", "0.005", "0.0013", "300", "5"]
```
Comment (asaj) [inline packages/protocol/releaseData/initializationData/release3.json]: Please explain in the PR description what's going on with StableToken being set to the zero address, and what the plan is

Comment (asaj) [inline packages/protocol/releaseData/initializationData/release3.json]: The spread is a fixidity value, please fix

Comment (asaj) [inline packages/protocol/releaseData/initializationData/release3.json]: The reserve fraction is a fixidity value, please fix

**Comment by asaj** [comment]
Please fill out the PR description template.

File: packages/protocol/releaseData/initializationData/release3.json (hunk 4 above)
Comment (martinvol) [inline packages/protocol/releaseData/initializationData/release3.json]:
```suggestion
  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", ["0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972", "0x989e1a3B344A43911e02cCC609D469fbc15AB1F1"], ["2000000000000000000000000", "6000000000000000000000000"], "ExchangeEUR"],
```

Comment (martinvol) [inline packages/protocol/releaseData/initializationData/release3.json]: 2M cEUR for the community fund at 0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972
6M cEUR to the foundation at 0x989e1a3B344A43911e02cCC609D469fbc15AB1F1

File: packages/protocol/releaseData/initializationData/release3.json [hunk 6]
```diff
@@ -1,4 +1,4 @@
 {
-  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", "0x0000000000000000000000000000000000000000", "0", "ExchangeEUR"],
-  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", "0.0005", "0.01", "300", "1"]
-}
\ No newline at end of file
+  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", ["0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972", "0x989e1a3B344A43911e02cCC609D469fbc15AB1F1"], ["2000000000000000000000000", "6000000000000000000000000"], "ExchangeEUR"],
```
Comment (martinvol) [inline packages/protocol/releaseData/initializationData/release3.json]:
```suggestion
  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1000000000000000000000000", "47304000", ["0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972", "0x989e1a3B344A43911e02cCC609D469fbc15AB1F1"], ["2000000000000000000000000", "6000000000000000000000000"], "ExchangeEUR"],
```

File: packages/protocol/releaseData/initializationData/release3.json [hunk 7]
```diff
@@ -1,4 +1,4 @@
...
+  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", "0x0000000000000000000000000000000000000000", "0.005", "0.0013", "300", "5"]
```
Comment (martinvol) [inline packages/protocol/releaseData/initializationData/release3.json]:
```suggestion
  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", "0x0000000000000000000000000000000000000000", "5000000000000000000000", "1300000000000000000000", "300", "5"]
```

File: packages/protocol/releaseData/initializationData/release3.json (hunk 4 above)
Comment (martinvol) [reply packages/protocol/releaseData/initializationData/release3.json]: addressed

File: packages/protocol/releaseData/initializationData/release3.json (hunk 5 above)
Comment (martinvol) [reply packages/protocol/releaseData/initializationData/release3.json]: addressed

**Review by yorhodes — APPROVED** [review]

**Pull request merged**